p4-16/psa/PSA.mdk

```diff
@@ -1109,8 +1106,10 @@ the contents of several metadata fields in the struct
     }
     // The value istd.egress_port below is the same one that the
     // packet began its egress processing with, as decided during
-    // ingress processing for this packet.  The egress code is not
-    // allowed to change it.
+    // ingress processing for this packet (or as determined by the PRE
+    // configuration of a clone session, for cloned packets,
+    // regardless of whether the clone operation was done in ingress
+    // or egress).  The egress code is not allowed to change it.
```

Comment:
On re-reading this, I noticed that it said egress_port was "as decided during ingress processing for this packet", and it bugged me when I realized that it could also be determined via initiating a clone egress-to-egress operation during egress processing.  That was true earlier, too, I just never noticed it before.  No semantic change in PSA requirements here -- just I hope a more full/correct comment.